CHANGES.rst

```diff
@@ -9,6 +9,9 @@ New Features
   - The ``PixelAperture.plot()`` method now returns a list of
     ``matplotlib.patches.Patch`` objects. [#923]
 
+- ``photutils.background``
+  - Improved the performance of ``Background2D._calc_coordinates`` by a factor of ~ 25. [#1103]
```

Comment:
```suggestion
- ``photutils.background``

  - Improved the performance of ``Background2D`` (e.g., by a factor
    of ~4 with 2048x2048 input arrays when using the default interpolator).
    [#1103]
```

Much better to report the overall performance improvement instead of that on a private method.